### PR TITLE
NL-64 Weekly Panel Backend

### LINKED ISSUE #50: Weekly Preview render goals
## Name and Email
  - Name: Chris Fellowes
  - Email: cfellowe@ucsd.edu

## Complexity of Issue (Scale from 1-5)
  - 3
  
## Issue Description
  - making the weekly goals dynamically rendered

### LINKED ISSUE #64: Connect Weekly Preview Panel to Backend
## Name and Email
  - Name: Rickesh Khilnani
  - Email: rikhilna@ucsd.edu

## Complexity of Issue (Scale from 1-5)
  - 3
  
## Issue Description
  - Going to edit WeeklyJS.js

### PR DESCRIPTION
**Pull request by Rick10101221**
Closes #64 

Add recursive function in WeeklyJS.js for rendering child lists

Create getCurrentWeek and getCurrentDay for rendering weekly data

Fix google auth button id in Login.js

### CODE REVIEW

**Comment by Rick10101221** [comment]
Closes #50

**Comment by Rick10101221** [comment]
MUST MERGE [PR 56](https://github.com/CSE112-sp22-group9/cse112-sp22-group9/pull/56) FIRST!

File: source/Backend/BackendInit.js [hunk 1]
```diff
@@ -23,7 +18,16 @@ getUserID().then((user) => {
  * @param {String} dayObj.notes - a string representing the notes
  * @returns void
  */
-function createDay(dayObj) {
+async function createDay(dayObj) {
+    const currentUserID = await getUserID()
```
Comment (DustinLin) [inline source/Backend/BackendInit.js:43]: Talked about adding this as a cookie, so we don't have call this getter every time. Perhaps in a next iteration

**Review by DustinLin — APPROVED** [review]

**Comment by Rick10101221** [comment]
Mergeable.

**Pull request merged**